.changeset/3298-custom-content-typing.md

```diff
@@ -0,0 +1,7 @@
+---
+"@ariakit/core": minor
+"@ariakit/react-core": minor
+"@ariakit/react": minor
+---
```

Comment:
Since we're still in v0, incrementing minor (`v0.x`) is dedicated to breaking changes. We should use `patch` for features and bug fixes until v1.
```suggestion
---
"@ariakit/core": patch
"@ariakit/react-core": patch
"@ariakit/react": patch
---
```